website/docs/docs/getting-started/2-your-first-test.md

```diff
@@ -0,0 +1,41 @@
+---
+id: your-first-test
+title: Writing your first test
+---
+
+To demonstrate how you can write tests, let's install the TodoMVC app from BigTest:
+```
+$ yarn add -D @bigtest/todomvc
```

Comment:
When we install this, is it to the project initiated by BigTest?

Reply:
I don't think bigtest is initiating a project. I think the assumption is people would be adding bigtest _to_ their pre-existing project.

So that being said, we're adding the todomvc app as a dev dependency (presumably to their pre-existing app) just so they can run bigtest against it. It's for demonstration purposes only and people should remove/delete it once they're done following along and understand how it works.

All they would need to do is remove the dependency of todomvc and modify the app command in `bigtest.json`.